examples/app_multi_node/app_pl_work.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import lightning as L
+from lightning.app.components import MultiNode
+from lightning.pytorch.demos.boring_classes import BoringModel
+
+
+class PyTorchLightningMultiNode(L.LightningWork):
```

Comment:
In order to reduce confusion, we could name this `PyTorchLightningDistributed`.

This way conceptually you pass a `Distributed`-aware PyTorchLightning script to a `MultiNode` orchestrator. Otherwise IMO it's kind of confusing to be passing a `MultiNode` something to another `MultiNode` something

Reply:
Done !

Reply:
how about naming it `LightningTrainerDistributed`?

Reply:
I prefer Luca proposition :)